args/args.go

```diff
@@ -112,7 +112,7 @@ func (g *GeneratorArgs) LoadGoBoilerplate() ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	b = bytes.Replace(b, []byte("YEAR"), []byte(strconv.Itoa(time.Now().Year())), -1)
+	b = bytes.Replace(b, []byte("YEAR"), []byte(strconv.Itoa(time.Now().UTC().Year())), -1)
```

Comment:
I guess the verification script / server is using UTC? This makes sense from that perspective, but it will be confusing for a user expecting their tools to use their local time zone.

Reply:
Travis does (as should all cloud servers); don't know about Prow. But as long as CI uses this same code, it will agree on which year should be in comments.